examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+import argparse
+import json
+from pathlib import Path
+from typing import Any, Tuple
+
+import cv2
+import numpy as np
+import numpy.typing as npt
+import rerun as rr
+import trimesh
+from download_dataset import AVAILABLE_RECORDINGS, ensure_recording_available
+from scipy.spatial.transform import Rotation as R
+from tqdm import tqdm
+
+# hack for now since dataset does not provide orientation information, only known after initial visual inspection
+ORIENTATION = {"48458663": "landscape", "42444949": "portrait", "41069046": "portrait"}
+assert len(ORIENTATION) == len(AVAILABLE_RECORDINGS)
+assert set(ORIENTATION.keys()) == set(AVAILABLE_RECORDINGS)
+
+
+def load_json(js_path: Path) -> dict[str, Any]:
+    with open(js_path, "r") as f:
+        json_data = json.load(f)
+    return dict(json_data)
+
+
+def log_annotated_bboxes(annotation: dict[str, Any]) -> None:
+    """Logs annotated bounding boxes to Rerun."""
+    for label_info in annotation["data"]:
+        object_id = label_info["objectId"]
+        label = label_info["label"]
+        rotation = np.array(label_info["segments"]["obbAligned"]["normalizedAxes"]).reshape(3, 3)
+        transform = np.array(label_info["segments"]["obbAligned"]["centroid"]).reshape(-1, 3)[0]
+        scale = np.array(label_info["segments"]["obbAligned"]["axesLengths"]).reshape(-1, 3)[0]
+
+        rot = R.from_matrix(rotation)
+        rr.log_obb(
+            f"world/annotations/box-{object_id}-{label}",
+            half_size=scale,
+            position=transform,
+            rotation_q=rot.as_quat(),
+            color=[160, 230, 130, 0],
+            label=label,
```

Comment:
Rather than log these with `label`, it would be nice to use `class_id` + annotation context (see for instance the example `tracking_hf_opencv`)

Reply:
So I tried looking at both the documentation on what is provided by arkitscenes and the tracking_hf_opencv example but it looks like only the `label` is provided to differentiate. There is this `object_id` but its the same for all of the different objects so I'm not totally sure what that's there for. For now what I did was remove the color so there could be more differentiation. I'm not fully sure if that's what you meant, but a bit more context would be helpful

Reply:
I also wanted to ask if you'd like me to reproject the 3d bounding boxes into the 2d camera view (similar to the objectron dataset) or if its better to wait for #1581

Reply:
> So I tried looking at both the documentation on what is provided by arkitscenes and the tracking_hf_opencv example but it looks like only the label is provided to differentiate.

Alright, makes sense. I just went off of the images in the original repo and assumed there was some kind of label map. Let's stick with just labels.

> I also wanted to ask if you'd like me to reproject the 3d bounding boxes into the 2d camera view (similar to the objectron dataset) or if its better to wait for https://github.com/rerun-io/rerun/issues/1581. For now I just added a todo regarding this

I think it's going to be a couple weeks at least before we add that proper so I'd manually reproject them and add a todo to remove the manual code once automatic reprojection is fixed